### PR TITLE
KIT-29 Fixed Test on data-race

### PR DESCRIPTION
**Pull request by GriffithBeliever**

### CODE REVIEW

**Review by jkrus — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+theme: jekyll-theme-architect
```
Comment (platsko) [inline _config.yml:1]: What is it?

File: strings/rand.go [hunk 2]
```diff
@@ -3,18 +3,18 @@
 package strings
 
 import (
-	"strings"
-
 	"github.com/evenlab/go-kit/bytes"
+	"strings"
```
Comment (platsko) [inline strings/rand.go:7]: plz make a fix by linter requirement.

**Review by platsko — CHANGES_REQUESTED** [review]

File: strings/rand.go (hunk 2 above)
Comment (jkrus) [reply strings/rand.go:7]: I have:
context/context.go:98:3: use of `print` forbidden by pattern `^(fmt\.Print(|f|ln)|print|println)$` (forbidigo)
what linter doesn't like about "print"

File: _config.yml (hunk 1 above)
Comment (jkrus) [reply _config.yml:1]: rudiment from the master))

Comment (platsko) [reply _config.yml:1]: Plz remove this file

File: strings/rand.go (hunk 2 above)
Comment (platsko) [reply strings/rand.go:7]: This message contains the name of the linter - forbidigo. You could just ignore it for now.

**Review by platsko — CHANGES_REQUESTED** [review]

**Pull request closed without merging**